easybuild/easyconfigs/j/jax/jax-0.2.18-foss-2020b.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'jax'
+version = '0.2.18'
+
+homepage = 'https://pypi.python.org/pypi/jax'
+description = """Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and
+more"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/google/jax/archive/']
+sources = ['%(name)s-v%(version)s.tar.gz']
+patches = [
+    'jax-0.2.18-correct_libraries.patch',
+]
+
+preinstallopts = 'python3 build/build.py '
+preinstallopts += '--bazel_startup_options="--output_user_root=%(builddir)s" '
+preinstallopts += ' --bazel_path="$EBROOTBAZEL/bin/bazel" && '
```

Bazel is in PATH and already checked. So maybe this instead:
```suggestion
preinstallopts += '--bazel_options="--subcommands --jobs=%(parallel)s" && '
```